requirements.txt

```diff
@@ -0,0 +1,2 @@
+nose
```

Comment:
Put `-c constraints.txt` on top of this file

Reply:
Would you believe me if I said I actually had `pytest` before? :smile: I have to say that I just find the simplicity of nose just nice still.

Reply:
:man_shrugging: